src/server.cpp

```diff
@@ -148,8 +148,8 @@ Server::Server(
 		const std::string &path_world,
 		const SubgameSpec &gamespec,
 		bool simple_singleplayer_mode,
-		bool ipv6
-	):
+		bool ipv6,
```

Comment:
Perhaps the compiler already does that, doesn't it?
Better we should have a complete struct with other stuff than just bools, like the MainMenuData struct.